packages/components/src/components/FormSelect/FormSelect.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import type { Meta } from "@storybook/react";
+import React from "react";
+import { useUID } from "react-uid";
+import type { SelectProps } from "../Select";
+import { FormSelect } from "./FormSelect";
+
+const meta: Meta<typeof FormSelect> = {
+  title: "Components/Form/FormSelect",
+  component: FormSelect,
+};
+
+export default meta;
+
+const selectItems: SelectProps["items"] = [
+  { value: "option-one", label: "Option One" },
+  { value: "option-two", label: "Option Two" },
+  { value: "option-three", label: "Option Three", disabled: true },
+  { value: "option-four", label: "Option Four" },
+  {
+    value: "option-five",
+    label:
+      "Option Five which is a really long item and should overflow to the next line",
+  },
+];
+
+export const Default = (): JSX.Element => {
+  const selectID = useUID();
+  return (
+    <FormSelect
+      id={selectID}
+      items={selectItems}
+      label="Label text"
+      name="select-field"
+    />
+  );
+};
+
+export const Required = (): JSX.Element => {
+  const selectID = useUID();
+  return (
+    <FormSelect
+      helpText="Please choose one of the values."
+      id={selectID}
+      items={selectItems}
+      label="Label text"
+      name="select-field"
+      required
+    />
+  );
+};
+
+export const Disabled = (): JSX.Element => {
+  const selectID = useUID();
+  return (
+    <FormSelect
+      disabled
+      helpText="Please choose one of the values."
+      id={selectID}
+      items={selectItems}
+      label="Label text"
+      name="select-field"
+    />
+  );
+};
+
+export const Error = (): JSX.Element => {
+  const selectID = useUID();
+  return (
+    <FormSelect
+      hasError
+      helpText="This is the error text."
+      id={selectID}
+      items={selectItems}
+      label="Label text"
+      name="select-field"
+    />
+  );
+};
+
+export const MultiSelect = (): JSX.Element => {
+  const selectID = useUID();
+  return (
+    <FormSelect
+      defaultValue={[selectItems[0].value, selectItems[1].value]}
+      helpText="Please choose one of the values."
+      id={selectID}
+      items={selectItems}
+      label="Label text"
+      name="select-field"
+    />
+  );
+};
```

Comment:
It might be cool to add an interaction test here to see the mouse click the select. But not essential. https://storybook.js.org/docs/7.0/react/writing-tests/interaction-testing

Reply:
Maybe something we could add to the Select component in the future.